mifosng-android/src/main/java/com/mifos/mifosxdroid/online/surveylist/SurveyListFragment.java

```diff
@@ -75,6 +75,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle
         rootView = inflater.inflate(R.layout.fragment_survey_list, container, false);
 
         ButterKnife.bind(this, rootView);
+        setToolbarTitle("Survey List");
```

Comment:
This should be `Surveys` and Please do not use hard coded strings. remove this string into strings.xml.